target_benchmark/generators/GeneratorPrompts.py

```diff
@@ -1,3 +1,7 @@
+## Language Models ##
```

Comment:
Not sure if this is the best place? I'd more expect this setting in a root-level config file.. But we can move it later!

Reply:
added an [issue](https://github.com/target-benchmark/target/issues/28#issue-2616774714) for later